docker/Dockerfile.gen-doc

```diff
@@ -0,0 +1,5 @@
+FROM --platform=$BUILDPLATFORM golang:1.23-bookworm
```

Comment:
Just a thought, why not create a single image to generate all the dependencies at once? I've split the `gen` commands to make them easier to fix, but I guess we'll run these commands all at once, either locally or in CI.

Reply:
This approach was my first solution, but I decided to split it so that we could run each command separately.